src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -434,10 +415,9 @@ public static Runspace CreateRunspace(PSHost psHost, PSLanguageMode languageMode
                 this.PromptContext = new LegacyReadLineContext(this);
             }
 
-            if (VersionUtils.IsWindows)
-            {
-                this.SetExecutionPolicy();
-            }
+            // Finally, restore the runspace's execution policy to the user's policy instead of
+            // Bypass.
+            this.RestoreExecutionPolicy();
```

Comment:
@rjmholt I moved the "if windows" to "if not windows" inside the function itself.

Reply:
@dkattan This still needs to be called because it restores the process scope policy when it was set on the CLI (such as by the extension).

Reply:
I think I learned this the hard way in my constrained runspace branch. I makes a lot more sense now that it is called RestoreExecutionPolicy.